clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/task/datawriter/AbstractPartitionWriter.java

```diff
@@ -576,7 +554,8 @@ protected void detectAndHandleDuplicateKeys(
           identicalValuesToKeyCount++;
           if (shouldPrint) {
             shouldPrint = false;
-            LOGGER.warn(printDuplicateKey(keyBytes));
```

Comment:
Do you think it's better to remove this, or is this just a shortcut as part of iterating on this change and you're planning to add it back after resolving some blocker...?

Reply:
The reason to remove this is it was failing due to configs missing. It is easy to have this impl, but I think it is not the right thing to log keys or values. Either of these can have private information that should not be logged. In addition to this, I have never seen a case where we relied on this to get the relevant info.

It is easy to use external analysis tools like Hive/Spark/Trino to get this info from the input data